package.json

```diff
@@ -1,5 +1,32 @@
 {
+  "name": "istanbuljs",
+  "version": "1.0.0",
```

Comment:
haha no worries. One more question though, will this increment in minor versions [only] for any changes to sub-packages, except breaking changes? That would make sense to me... though I'm not entirely sure.

Or do we want to go with the babel versioning system and start at v10 [i.e. like nyc] and increment everything together?